geo/src/algorithm/area.rs

```diff
@@ -107,18 +135,27 @@ impl<T> Area<T> for MultiPolygon<T>
 where
     T: Float,
 {
-    fn area(&self) -> T {
+    fn signed_area(&self) -> T {
         self.0
             .iter()
-            .fold(T::zero(), |total, next| total + next.area())
+            .fold(T::zero(), |total, next| total + next.signed_area())
+    }
+
+    fn unsigned_area(&self) -> T {
+        self.signed_area().abs()
```

Comment:
hm... is this correct?

e.g. if I had 
```
polygon_1.signed_area == 1 
polygon_2.signed_area == -1
```

and
```
m = MultiPolygon::new(polygon_1, polygon_2)
```

then shouldn't `m.unsigned_area == 2`?

But the current code would give `m.unsigned_area == 0`

Should we instead sum each polygon's unsigned area like you did with GeometryCollection?

Reply:
@michaelkirk great catch 🏆 

fixed in https://github.com/georust/geo/pull/463/commits/a030e0b9cf67a0b9b87e3fcc55e23766511d5a02